org.openscad.OpenSCAD.json

```diff
@@ -0,0 +1,164 @@
+{
+    "app-id": "org.openscad.OpenSCAD",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.10",
+    "sdk": "org.kde.Sdk",
+    "command": "openscad",
+    "rename-appdata-file": "openscad.appdata.xml",
```

Comment:
Upstreams appstream file is missing info and will need to be improved:

- Missing license
- Missing releases
- Old format